torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""CropHarvest datasets."""
+
+import glob
+import json
+import os
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import download_and_extract_archive, download_url
+
+
+class CropHarvest(NonGeoDataset):
+    """CropHarvest dataset.
+
+    The CropHarvest <https://github.com/nasaharvest/cropharvest> datataset is a
+    classification dataset.
```

Comment:
```suggestion
    classification dataset for crop classification.
```
or something like that, add a little more detail about the intended use of the dataset